src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBlockTests/TextBlockTests.cs

```diff
@@ -55,5 +55,30 @@ public void When_Multiple_Controls_And_Some_Are_Collapsed_Arrange()
 				left.Y.Should().Be(right.Y, "Y position");
 			}
 		}
+
+		[Test]
+		[AutoRetry]
+		public void When_Foreground_Changed_With_Visibility()
```

Comment:
This test is not executed here: https://github.com/unoplatform/uno/blob/71a779dda1ac0fedf187e06a7d5a749599b26ab2/build/ios-uitest-run.sh#L30

Reply:
Indeed!

I'll whitelist the `TextBlockTests` and individually disable any that fail.